src/Groups/cosets.jl

```diff
@@ -65,13 +74,72 @@ function show(io::IO, x::GroupCoset)
    a = GAP.gap_to_julia(GAP.Globals.StringView(x.H.X))
    b = GAP.gap_to_julia(GAP.Globals.StringView(x.repr.X))
    if x.side == :right
-      print(io, a, " * ", b)
+      print(io, "Right coset   ", a, " * ", b)
    else
-      print(io, b, " * ", a)
+      print(io, "Left coset   ", b, " * ", a)
    end
    return nothing
 end
 
+"""
+    isleftcoset(c::GroupCoset)
+Return whether the coset `c` is a left coset of its acting domain.
+"""
+isleftcoset(c::GroupCoset) = c.side == :left
```

Comment:
I think we should follow Tommy's suggestion and call this `isleft` (and the other `isright`)

Reply:
Done.